src/c++/perf_analyzer/raw_data_reporter.h

```diff
@@ -0,0 +1,75 @@
+// Copyright 2023, NVIDIA CORPORATION & AFFILIATES. All rights reserved.
```

Comment:
I'm not seeing any unit testing for these new classes.

Reply:
@nv-hwoo is working on this and it will be in a separate PR 👍